plugins/inputs/http/http_test.go

```diff
@@ -488,6 +488,9 @@ func TestConnectionOverUnixSocket(t *testing.T) {
 		}
 	}))
 
+	// The Maximum length of the socket path is 104/108 characters, path created with t.TempDir() is too long for some cases
+	// (it combines test name with subtest name and some random numbers in the path)
+	//nolint:usetesting // "os.TempDir() could be replaced by t.TempDir() in TestConnectionOverUnixSocket"
 	unixListenAddr := filepath.Join(os.TempDir(), fmt.Sprintf("httptestserver.%d.sock", rand.Intn(1_000_000)))
```

Comment:
I don't see a removal of the temporary directory...

Reply:
Same as above

Reply:
These are just globs used by plugin.
Socket files created later are automatically removed after socker is closed.